rosdep/python.yaml

```diff
@@ -2672,6 +2672,18 @@ python-pytest:
   fedora: [python-pytest]
   gentoo: [dev-python/pytest]
   ubuntu: [python-pytest]
+python-pytest-qt:
```

Comment:
We can remove that key altogether as it doesn't have any resolution for the platforms we require support for.

We will consider adding it back once it's available for all platforms targeted by at least one ROS distribution

Reply:
sure